src/parser.test.js

```diff
@@ -0,0 +1,90 @@
+import parser from './parser';
+import { renderIntoDocument } from '../tests/utils/dom-render';
+
+const setup = () => {
+  return renderIntoDocument(`
+    <div>
+      <button>btn1</button>
+      <button>btn2</button>
+      <span>I'm a label</span>
+      <input placeholder="I'm a placeholder" />
+    </div>`);
+};
+describe('parser', () => {
```

Comment:
No need for this `describe`, since the filename's already saying we're testing `parser`